src/components/SubTypeAccordion/SubTypeAccordionsContainer.js

```diff
@@ -7,15 +7,14 @@ import { shape, bool } from 'prop-types';
 
 import SubTypeAccordion from './SubTypeAccordion';
 
-const SubTypeAccordionsContainer = ({ subTypeData, fromContentPanel }) => {
-  if (subTypeData === {}) {
-    return null;
-  }
```

Comment:
^this removed `if` condition/block will never evaluate as `true`:
<img width="196" alt="image" src="https://user-images.githubusercontent.com/3383704/113038857-0239d100-9165-11eb-851c-2aed17ef2232.png">